.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
generally you don't need .vscode settings unless you want to share them with the team.
you can add .vscode to the git ignore file so you don't need to remove it from your commits every time.